added new options: api, start-on-load (#103)
ain/smartbanner.js: README.md

<meta name="smartbanner:hide-path" content="/">
```

> ### Set "smartbanner" as global variable

Reviewer: ```suggestion
### smartbanner API use
```

> By default smartbanner not support manual construction. You can enable it and use "window.smartbanner" variable

Reviewer: ```suggestion
By default smartbanner is added to DOM automatically. You can disable it with
```

```html
<meta name="smartbanner:api" content="true">
```

> **Note:** By default smartbanner initialise on window load. But when used option *api* you should initialise smartbanner manually 

Reviewer: ```suggestion
and add smartbanner to DOM manually:
```

```js

> smartbanner.publish()

Reviewer: ```suggestion
smartbanner.publish();
```

<meta name="smartbanner:hide-path" content="/">
```

### Set "smartbanner" as global variable

By default smartbanner not support manual construction. You can enable it and use "window.smartbanner" variable

```html
<meta name="smartbanner:api" content="true">
```

> 

Reviewer: Whenever API is used, publishing should also be a responsibility of a developer which would make `start-on-load` obsolete. I'd support single option here, the API.

Author: Removed obsolete `start-on-load`
